app-api/src/main/java/org/openmrs/eip/app/management/entity/ReceiverSyncStatus.java

```diff
@@ -2,19 +2,21 @@
 
 import java.util.Date;
 
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 @Entity
 @Table(name = "receiver_sync_status")
-public class ReceiverSyncStatus extends AbstractEntity {
-	
+public class ReceiverSyncStatus extends AbstractEntity {	
 	public static final long serialVersionUID = 1;
 	
-	//TODO This should be mapped as a @OneToOne association when we build a UI
-	@Column(name = "site_info_id", nullable = false, updatable = false, unique = true)
-	private Long siteInfoId;
+	@OneToOne(cascade = CascadeType.ALL)
+	@JoinColumn(name = "site_info_id", referencedColumnName = "id")
```

Comment:
The referenceColumnName value defaults to the primary key column so setting it here is  redundant

Reply:
I see